components/surveyBox/SurveyBox.module.css

```diff
@@ -121,6 +118,40 @@
   align-self: stretch;
   flex-grow: 0;
 }
-.line th {
-  padding-right: 135px;
+
+.surveyName{
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0px;
+
+    width: 400px;
+    height: 24px;
+    /* Inside auto layout */
+
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+}
+.prerequisites{
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0px;
+
+    width: 200px;
+    height: 24px;
+    /* Inside auto layout */
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+}
+.endDate{
+
+}
+.rewardAmount{
+
+}
+.progress{
+
 }
```

Comment:
new line is needed at the at the end of the file
ref.: [whats-the-point-in-adding-a-new-line-to-the-end-of-a-file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)